communications/GSOD-applications/2020-gsod.md

```diff
@@ -0,0 +1,57 @@
+# Google Season of Docs (GSOD) 2020
+
+* Website: https://developers.google.com/season-of-docs
+* Location: Distributed/online
+* Date: 10-11 March 2020
+* Context: Technical writers spend three months working closely with an open source community and learn about open source and new technologies. 
+* INCF call for the organisation partners: https://www.incf.org/blog/call-mentors-join-us-gsod-2020
+* Previous year’s application: https://github.com/alan-turing-institute/the-turing-way/issues/406
+
+### Title
+
+* The Turing Way project expands into a series of books on data research practices
+
```

Comment:
Whats happening with the formatting here? Is the title that long?

Reply:
Sorry - I meant to go back and rephrase this - I think the title is supposed to capture the expansion but I don't think it's necessary. Just "The Turing Way, a how to guide to data science" would work I think?